api/src/utils/validation.ts

```diff
@@ -10,3 +10,37 @@ import { ObjectId } from 'mongodb';
  */
 export const isObjectID = (id?: string): boolean =>
   id ? ObjectId.isValid(id) : false;
+
+// Refer : http://stackoverflow.com/a/430240/1932901
+/**
+ * Sanitizes a input by removing HTML tags.
+ * @param value A string to sanitize.
+ * @returns A string with HTML tags removed.
+ */
+export const trimTags = (value: string): string => {
```

Comment:
I know this function has been passed around for years, but if you've got a moment to create some tests it's an obvious candidate!

Reply:
I'll add the tests for this function in a separate PR